java/1.2/modules/ROOT/pages/configure-java-module-acb.adoc

```diff
@@ -0,0 +1,306 @@
+
+== Before You Begin
+
+Before creating an Anypoint Code Builder integration project, you must:
+
+* xref:anypoint-code-builder::start-acb.adoc[Set up and access the Anypoint Code Builder web or desktop IDE]
+* Have credentials to access Anypoint Platform
+* Ensure that the Visual Studio Code Extension Pack for Java is installed.
+
+[[creating-new-integration-project]]
+== Creating a New Integration Project
+
+include::connectors::partial$configure-acb-connectors-partial.adoc[tag="acb-create-integration-project"]
+
+When you create a new Anypoint Code Builder project, `src/main/java` and `/src/test/java` folders are created. For existing projects, you can manually create this folder. 
+
+
+
+[[add-components]]
+== Add Components to Your Project
+
+Add components to your integration project to build a flow:
+
+. In the *Explorer* view, open the XML file for your project if it's not already open.
+. Select *Build a Flow* from the start card to create an empty flow:
++
+image:connectors::acb-build-flow.png["Canvas UI with starting card with the option to start from scratch, build a flow, or build a subflow"]
++
+. Change the name of the flow: 
+.. Click *Flow*. 
+.. Click the edit icon. 
++
+image:connectors::acb-change-flow-name.png["Arrow pointing to the flow name edit pencil icon"]
++
+.. Enter the flow name, and click the checkmark. 
+
+[[configure-trigger]]
+== Configure a Trigger for the Flow
+A trigger (source) starts the flow when specified conditions are met. If a connector has its own associated sources, you can use a connector-specific trigger to initiate the flow. Each connector trigger has a unique configuration.
+
+You can configure the following sources for this connector:
+
+include::connectors::partial$t-studio-partial.adoc[tags=listener-scheduler-descriptions]
+
+[[adding-components-to-project]]
+== Adding Components to Your Project
+
+include::connectors::partial$configure-acb-connectors-partial.adoc[tag="acb-add-components-to-project"]
+
+[[configuring-component-attributes]]
+== Configuring the Component Attributes
+
+include::connectors::partial$configure-acb-connectors-partial.adoc[tag="acb-configure-attributes"]
+
+[[configuring-new]]
+=== Configuring the New Operation
+
+Create a new instance of a class by using the *New* operation of the Java module. The identifier of a constructor includes the class and constructor names, and the constructor has a full description of its signature including the types of each parameter.
+
+For example, to invoke the constructor `Me(String name, int age)` which belongs to the `org.bar.Me` class, then the identifier of the method is `Me(String,int)`.
+
+To configure the attributes for *New* in the Anypoint Code Builder UI:
+
+. Click the *New* component on the canvas.
+. In the *General* tab:
+.. Select the *Class* that contains the method to use.
+.. Select the *Method* to use. 
```

Comment:
maybe not needed here, but there's a KI on selecting the method after changing classes -- you have to click refresh to see the list of methods in the new class

perhaps could say something future proof like, "If there is an issue with the method list, refresh the field." not great wording but whatever. Or we can assume they'll try that without prompting.

Reply:
not that they're changing classes in this case, so... whatever you think is best